File: raguilar6.c
//Ryan Aguilar
//CSC321
//Lab 6b

#include <stdio.h>
#include <stdlib.h>
#include <math.h>

int main(void)
{
	int x = 5;
	int y = 10;
	int answer = 0;

	answer = x + y * x / y - x;
	printf("Answer to first expr = %i\n", answer);

	answer = - x - y / x * y + x;
	printf("Answer to second expr = %i\n", answer);

	answer = x + y - x / y;
	printf("Answer to third expr = %i\n", answer);
}	

File: raguilar6.java
//Ryan Aguilar
//CSC321
//Lab 6b

public class raguilar6
{
	
	public static void main(String[] args)
	{
		int x = 5;
		int y = 10;
		int answer;

		answer = x + y * x / y - x;
		System.out.println("Answer to first expr = " + answer);

		answer = -x - y / x * y + x;
		System.out.println("Answer to the second expr = " + answer);

		answer = x + y - x / y;
		System.out.println("Answer to the third expr = " + answer);
	}
}
